aea/crypto/ledger_apis.py

```diff
@@ -145,6 +147,17 @@ def ethereum_api(self) -> EthereumApi:
         assert self.has_ethereum, "Ethereum API not instantiated!"
         return cast(EthereumApi, self.apis[ETHEREUM])
 
+    @property
+    def has_cosmos(self) -> bool:
+        """Check if it has the cosmos API."""
+        return COSMOS in self.apis.keys()
+
+    @property
+    def cosmos_api(self) -> CosmosApi:
+        """Get the Cosmos API."""
+        assert self.has_cosmos, "Cosmos API not instantiated!"
+        return cast(CosmosApi, self.apis[COSMOS])
+
```

Comment:
I think this pattern `has_x`/`x_api` should be improved. This design is not scalable in terms of the number of ledger APIs.

Reply:
I agree, should be changed.